test/JDBC/expected/sys-system_sql_modules-dep-vu-verify.out

```diff
@@ -4,8 +4,8 @@ GO
 EXEC sys_system_sql_modules_dep_vu_prepare_p1
 GO
 ~~START~~
-bit
-1
+bit#!#nvarchar
+1#!#
```

Comment:
why is definition empty here? same for sys_system_sql_modules_dep_vu_prepare_v1?

Reply:
We don't support definition of system objects. That would be NULL.